config/config.exs

```diff
@@ -174,7 +174,9 @@ config :omg_watcher_info,
   child_chain_url: "http://localhost:9656",
   namespace: OMG.WatcherInfo,
   ecto_repos: [OMG.WatcherInfo.DB.Repo],
-  metrics_collection_interval: 60_000
+  metrics_collection_interval: 60_000,
+  pending_block_processing_interval: 1000,
+  block_queue_check_interval: 10_000
```

Comment:
There's no particular reason why these two constants live in a configuration, right? They're not reused or available to configuration from outside. They could live in the module itself.

Reply:
It helps with seeing an overview of all the rhythms in the system though!